tests/test_db.py

```diff
@@ -7,6 +7,25 @@
 
 
 class ProtectedModelMixinTestCase(TestCase):
+    def test_update_protected(self):
+        protected = ProtectedModel(name="protected", is_update_protected=True)
+        protected.save()
+        with self.assertRaises(PermissionDenied):
+            protected.save()
```

Comment:
Why not use 
```suggestion
        with self.assertRaisesMessage(PermissionDenied, "This object has update protection."):
            protected.save()
```

and leave out `test_get_protected_against_update_message`?

Reply:
Atomic testing, `get_protected_against_update_message` is a public method, should be tested separately.

Reply:
ok, fair enough!